menyer/pizza_house/pizza_house.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<html lang="sv">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Pizza House</title>
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="./assets/pizza_house.css">
+</head>
+<body>
+
+    <main id="main">
+        
+        <div class="information">
+            <div class="column">
+                <h1>Papa John's</h1><hr>
+
+                <h2><i class="fa fa-phone"></i> 0763 123 456</h2>
+                <h2><i class="fa fa-clock-o"></i> Måndag-Söndag | 14.00-23.00</h2>
+            </div>
+            
+            <div class="column">
+                <img src="https://upload.wikimedia.org/wikipedia/en/thumb/c/c4/Papa_John%27s_Pizza_logo.svg/640px-Papa_John%27s_Pizza_logo.svg.png" alt="">
+            </div>
+        </div>
+
+
+        <br>
```

Comment:
`br` not needed.  My guess is that you used it more for sectioning your html. I recommend using a semantic element for this instead like `section`.